Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -628,15 +628,18 @@ def _makeKerningLookup(self, name, ignoreMarks=True):
                 )
         return lookup
 
-    def knownScriptsPerCodepoint(self, uv: int) -> set[str]:
+    def knownScriptsPerCodepoint(self, uv: int) -> Union[set[str], None]:
```

Comment:
`Union[T, None]` is more often called `Optional[T]`, or in python 3.10+ you can do:

```suggestion
    def knownScriptsPerCodepoint(self, uv: int) -> set[str] | None:
```